cordova-lib/package.json

```diff
@@ -55,7 +55,7 @@
     "test": "npm run jshint && npm run jasmine",
     "test-ios": "npm run test && npm run jasmine-ios",
     "ci": "npm run jshint && npm run cover && codecov",
-    "jshint": "jshint src spec-cordova spec-plugman",
+    "jshint": "./node_modules/.bin/jshint src spec-cordova spec-plugman",
```

Comment:
not sure if full path to jshint is needed anymore. That error you were getting earlier was solved by deleting browser fixture's tests